azurerm/resource_arm_automation_credential.go

```diff
@@ -52,10 +52,12 @@ func resourceArmAutomationCredential() *schema.Resource {
 
 func resourceArmAutomationCredentialCreateUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ArmClient).automationCredentialClient
+	ctx := meta.(*ArmClient).StopContext
 	log.Printf("[INFO] preparing arguments for AzureRM Automation Credential creation.")
 
 	name := d.Get("name").(string)
 	resGroup := d.Get("resource_group_name").(string)
+	client.ResourceGroupName = resGroup
```

Comment:
We could probably refactor the client initialisation to handle the repetition (similar to how we have with Storage) - but we can do that at a later date